tobac/utils/general.py

```diff
@@ -553,3 +554,129 @@ def combine_tobac_feats(list_of_feats, preserve_old_feat_nums=None):
 
     combined_df = combined_df.reset_index(drop=True)
     return combined_df
+
+
+@internal_utils.irispandas_to_xarray
+def transform_feature_points(
+    features,
+    new_dataset,
+    latitude_name="auto",
+    longitude_name="auto",
+    max_time_away=datetime.timedelta(minutes=1),
+    max_space_away=20000,
```

Comment:
I'd prefer these set to `None` (or `np.inf`) by default rather than fixed values. The fixed values could lead to unexpected behaviour for some users (e.g. if they have data with low spatial/temporal resolution)

Reply:
I've set these to a default value of 0 and 0.